includes/discovery/sensors/temperatures/serveriron.inc.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+if ($device['os'] == 'serveriron') {
+    echo(" FOUNDRY-SN-AGENT-MIB ");
+
+    // Chassis temperature (default)
+    $high_limit		= 110;
+    $high_warn_limit	= 95;
+
+    $descr    = "Chassis Temperature";
+    $oid      = "1.3.6.1.4.1.1991.1.1.1.1.18.0"; // snChasActualTemperature
+    $warn_oid = "1.3.6.1.4.1.1991.1.1.1.1.19.0"; // snChasWarningTemperature
+    $high_oid = "1.3.6.1.4.1.1991.1.1.1.1.20.0"; // snChasShutdownTemperature
+    $value    = snmp_get($device, $oid, '-Oqv', 'FOUNDRY-SN-AGENT-MIB');
```

Comment:
I can fix that. Either put the symbolic name or remove the MIB.
What do you mean by "it is done multiple times"?

Regards